packages/ckeditor5-find-and-replace/tests/findandreplace.js

```diff
@@ -116,16 +116,6 @@ describe( 'FindAndReplace', () => {
 		describe( 'mocks', () => {
 			// Verifying mocks from https://github.com/ckeditor/ckeditor5/issues/9719#issuecomment-857557024.
 			it( 'has a proper initial state', () => {
-				// "Initial state" mock.
-				editor.setData( LONG_TEXT );
-
-				const itemView = Array.from( editor.ui.view.toolbar.items )
-					.filter( item =>
-						item.buttonView && item.buttonView.tooltip == 'Find and replace'
-					)[ 0 ];
-
-				itemView.buttonView.arrowView.fire( 'execute' );
```

Comment:
Why was this preparation logic removed?

This test was meant to test initial state after the dropdown is shown. With this change we check just state after formView was constructed but not really shown.

Reply:
This question is valid for other instances like that.

Reply:
You are right, that logic should've stayed there. I've reverted the changes made before.